lib/p2p/NodeList.ts

```diff
@@ -39,10 +40,26 @@ class NodeList extends EventEmitter {
   /**
    * Check if a node with a given nodePubKey exists.
    */
-  public has = (nodePubKey: string) => {
+  public has = (nodePubKey: string): boolean => {
     return this.nodes.has(nodePubKey);
   }
 
+  public remove = (nodePubKey: string): void => {
```

Comment:
Do we need this method? It doesn't look like it's being used and I don't think we'd want to remove/forget about nodes in this way.